app/src/main/res/values/styles.xml

```diff
@@ -19,18 +19,32 @@
         <item name="colorControlNormal">@color/colorControlNormal</item>
         <item name="android:textColorHint">@color/colorPlainText</item>
 
-        <item name="android:listPopupWindowStyle">@style/AppTheme.ListPopupWindow</item>
-    </style>
-
-    <style name="AppTheme.ListPopupWindow" parent="@style/Widget.AppCompat.ListPopupWindow">
-        <item name="android:popupBackground">@color/colorPrimaryDark</item>
+        <item name="android:listPopupWindowStyle">@style/ListPopupWindow</item>
     </style>
 
     <style name="AppTheme.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
 
+    <style name="AppTheme.NoActionBar.SamplePreparationActivity">
+        <item name="switchStyle">@style/Switch</item>
+    </style>
+
+    <style name="AppTheme.NoActionBar.MicroscopeQualityActivity">
+        <item name="switchStyle">@style/Switch</item>
+    </style>
```

Comment:
Hmm, why having two new identical styles for the new activities? Couldn't the `switchStyle` be added to the parent style/theme?

Reply:
Yep, definitely... I just tried to keep the styles as flexible and independent as possible, as I'm not sure where we're going regarding styling. I'd like, in fact, to not need to create any major styling customisation (which I think is the case here). Anyway, do you get my point? What do you think would be healthier regarding this?

Reply:
> flexibile

Yes, please.

> independent

I think that this is counter-productive when it comes to styles, as it leads to a lot of accidental variation throughout the app. But normally, you want consistency in the design of your app. 

So I would say to consolidate the main theme of the app and use just that, instead of creating additional themes. Anytime something needs to have a different style, we can do it per view or activity/view group level easily.

Reply:
Done, thanks!